### PR TITLE
Have the broker accept an undefined SC in its configuration and pass it through to the PVC.

### PR DESCRIPTION
**Pull request by andreas-kupries**
See https://github.com/cloudfoundry-incubator/kubecf/issues/819

By not specifying an SC we can make use of the DefaultSC admission plugin
to get something sensible in a more complex deployment with many helm charts
and sub charts.

# Testing

This pull request is one in a group of 3 related PRs.
They have to be reviewed and tested together.
The instructions for such are found at https://github.com/cloudfoundry-incubator/kubecf/issues/819#issuecomment-649841863
The full set of PRs is listed in the description of https://github.com/cloudfoundry-incubator/kubecf/issues/819

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ type Plan struct {
 	ID           string `yaml:"plan_id"`
 	Name         string `yaml:"plan_name"`
 	Description  string `yaml:"description"`
-	StorageClass string `yaml:"kube_storage_class"`
+	StorageClass *string `yaml:"kube_storage_class"`
```
Comment (mook-as) [inline config/config.go:45]: Hmm, Did `go fmt` not run correctly? (I'd have expected the `yaml`s to line up.)

**Review by mook-as — APPROVED** [review]
LGTM, but please double check that `go fmt` has completed successfully.

Though, we should probably add / modify a test to make sure we _do_ set it to `nil` by default.

File: config/config.go (hunk 1 above)
Comment (andreas-kupries) [reply config/config.go:45]: My go 1.13 always fails with 
> go: k8s.io/client-go@v2.0.0-alpha.0.0.20181213151034-8d9ed539ba31+incompatible: invalid pseudo-version: preceding tag (v2.0.0-alpha.0) not found

I managed to build using `docker build .`, which explicitly uses `go 1.12`.
Ok, managed to run indirectly by inserting `go fmt -x` into the dockerfile.
And then used `gofmt` directly, as reported by the `-x` option above.

Two files got reported, one is my changes, the other seems to have existed for longer.

Done with commit 6faa8cade3aac25f762d8ecad39423a760d71875
